### DISCUSSION TITLE
Are there plans to support other languages?

### DISCUSSION THREAD

**Discussion by echoowall** [teslamate-org/teslamate#1610] [Ideas]
Are there plans to support other languages?
Such as Chinese. The project has been used by users in many languages around the world.

**Comment by DrMichael** [comment]
Well, teslamate supports several languages. But Grafana unfortunately does not (yet) support localization.

**Reply by 582911658** [reply]
Grafana support Chinese  now. Are there plans to support Chinese in the future？

**Comment by star2k** [comment]
> Well, teslamate supports several languages. But Grafana unfortunately does not (yet) support localization.
> 
> 

Hello, don't you have simplified Chinese yet? I hope to add it as soon as possible.

**Comment by DrMichael** [comment]
Thinking about it... Grafana supports languages since 9.3.
@JakobLichterfeld Shouldn't we set _default_language_ to _detect_ in _/etc/grafana/grafana.ini_?
Still does nto help for localisation of the dashboards and panel strings itselves...

**Reply by JakobLichterfeld** [reply]
@swiffer and @oivindoh Do you have experience with Grafana localization? ( I don't)

**Reply by swiffer** [reply]
no experience so far.. i don't see any support for localization within the dashboard json

**Reply by swiffer** [reply]
we might can built it via dashboard variables and sql but that seems to be a hacky solution not scaling well with dashboards edits i guess

**Comment by thokaro** [comment]
Hi, were can i change the default language?

**Reply by JakobLichterfeld** [reply]
In Webpage, Settings Language.
Grafana does not support localisation.
